.github/workflows/ci-python.yml

```diff
@@ -15,7 +15,7 @@ on:
     branches:
       - master
     paths:
-      - '.github/**'
+      - '.github/workflows/ci-python.yml'
       - 'extract/**'
       - 'tests/**'
```

Comment:
I have added airflow_utils tests under tests/dags/*.

So for airflow, each test will be inside specific directory. like dags/tests/* etc and for rest its under tests/* ?.

Reply:
I checked the tests for plugins and noticed they are under `plugins/tests`. This is a good approach, as it helps separate the test code for python scripts from airflow. Having said that, I'd expect all tests for dags to be under `dags`. Pytest's documentation also discusses [different layouts for tests](https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#choosing-a-test-layout-import-rules).